hardhat.config.ts

```diff
@@ -9,4 +46,18 @@ export default {
   solidity: {
     version: "0.6.12",
   },
+  networks: {
+    mainnet: {
+      ...sharedNetworkConfig,
+      url: "https://mainnet.infura.io/v3/".concat(infuraKey),
```

Comment:
nit: Is there a reason you don't prefer format strings
```suggestion
      url: `https://mainnet.infura.io/v3/${infuraKey}`,
```

Reply:
No reason, I didn't think about it! Your suggestion looks better.